fulp_modules/features/clothing/halloween/costumes_2021/halloween_2021.dm

```diff
@@ -25,9 +25,122 @@
 	icon_state = "lady_maria_hat"
 
 /obj/item/storage/box/halloween/edition_21/oldhunter
-	theme_name = "Old Hunter's Set"
+	theme_name = "2021's Old Hunter's Set"
 	illustration = "Mask"
 
 /obj/item/storage/box/halloween/edition_21/oldhunter/PopulateContents()
 	new /obj/item/clothing/suit/oldhunter(src)
 	new /obj/item/clothing/head/oldhunter(src)
+
+// SHOVEL KNIGHT
+
+/obj/item/clothing/suit/spectre_suit
+	name = "Spectre knight's robes"
+	desc = "A darkened blood red robe with little armour."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_worn.dmi'
+	icon_state = "spectre_suit"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS|FEET
+	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+
+/obj/item/clothing/head/spectre_head
+	name = "Spectre knight's helmet"
+	desc = "A blood red hood, attatched to a reflective gold visor."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_worn.dmi'
+	icon_state = "spectre_head"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+/obj/item/clothing/suit/hooded/propeller_suit
+	name = "PropellerSuit"
```

Comment:
Why is this one word and why is it the propeller head not similarly capitalized

Reply:
idk i did it on different days